src/v/cluster/drain_manager.cc

```diff
@@ -40,35 +40,41 @@ ss::future<> drain_manager::stop() {
 }
 
 ss::future<> drain_manager::drain() {
```

Comment:
please update comments in the header file saying that this must be called on every shard